internal/clusterinfo/clusterinfo_test.go

```diff
@@ -0,0 +1,52 @@
+package clusterinfo
```

Comment:
Seems we could combine this test with the other since its supposed to be the same result with a different client.  Can we just add more entries and slightly modify it

Reply:
The static code & tests are uncompilable within the CLO module because of dependency clashes. I built them as a separate module to make sure the dynamic code was correct, but I'll remove now as they're a distraction.